tests/unit/utils.test.js

```
@@ -714,34 +730,55 @@ test('slugifyTagForFilePath should create filesystem-safe tag names', () => {
 
 	// Null/undefined tags should use base path
-	expect(getTagAwareFilePath(basePath, null, projectRoot)).toBe(
-		'/test/project/.taskmaster/reports/complexity-report.json'
-	);
+	result = getTagAwareFilePath(relativeBasePath, null, projectRootAbs);
+	expect(result).toBe(expected); // Same expected path as 'master'
 
 	// Regular tag should be slugified
-	expect(getTagAwareFilePath(basePath, 'feature-branch', projectRoot)).toBe(
-		'/test/project/.taskmaster/reports/complexity-report_feature-branch.json'
+	let parsedBasePath = path.parse(relativeBasePath);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'parsedBasePath' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)